server/views/pages/overview/partials/overviewTab/overviewTabContents.njk

```diff
@@ -44,9 +44,90 @@ Data supplied to this template:
       {% include './_educationAndTrainingSummaryCard.njk' %}
     </div>
 
+    {% set dueOrOnTimeTagContent = 'Review due by ' ~ reviewDateTo | formatDate('D MMM YYYY') %}
+    {% set dueOrOnTimeTagTitle = '<span>' ~ dueOrOnTimeTagContent ~ '</span>' %}
+    {% set overdueTagContent = 'Review was due by ' ~ reviewDateTo | formatDate('D MMM YYYY') %}
+    {% set overdueTagTitle = '<span>' ~ overdueTagContent ~ '</span>' %}
+    {% set releaseDateContent = 'Release on ' ~ releaseDate | formatDate('D MMMM YYYY') %}
+    {% set releaseDateTitle = '<span>' ~ releaseDateContent ~ '<span>' %}
+
     <div class="govuk-grid-column-one-third app-u-print-full-width">
       {{ actionsCard({
         actions: [
+          {
+            title: '<h2 class="govuk-heading-s govuk-!-margin-bottom-0">Reviews</h2>' | safe,
+            'render-if': induction.isPostInduction,
+            notAUrl: true
+          },
+           {
+            title: dueOrOnTimeTagTitle | safe,
+            'render-if': induction.isPostInduction and reviewStatus === 'onTime',
+            notAUrl: true,
+            border: true,
+            classes: 'actions-card-reviews-tag-grey',
+            attributes: {
+              'data-qa': 'review-on-time'
+            }
+          },
+           {
+            title: dueOrOnTimeTagTitle | safe,
+            'render-if': induction.isPostInduction and reviewStatus === 'due',
+            notAUrl: true,
+            border: true,
+            classes: 'actions-card-reviews-tag-yellow',
+            attributes: {
+              'data-qa': 'review-due'
+            }
+          },
+          {
+            title: overdueTagTitle | safe,
+            'render-if': induction.isPostInduction and reviewStatus === 'overdue',
+            notAUrl: true,
+            border: true,
+            classes: 'actions-card-reviews-tag-red',
+            attributes: {
+              'data-qa': 'review-overdue'
+            }
+          },
+           {
+            title: 'No reviews due',
+            'render-if': induction.isPostInduction and reviewStatus === 'noReviewsDue',
+            notAUrl: true,
+            classes: 'govuk-tag govuk-tag--grey',
+            attributes: {
+              'data-qa': 'no-reviews-due'
+            }
+          },
+           {
+            title: releaseDateTitle | safe,
+            'render-if': induction.isPostInduction and reviewStatus === 'noReviewsDue',
+            notAUrl: true,
+            heavyBorder: true
+          },
+          {
+            title: '<span class="goal-action"><img src="/assets/images/icon-mark-review-complete.svg" role="presentation" alt="" width="30px" height="24px" class="goal-icon" />Mark review as complete</span>' | safe,
```

Comment:
The more I think about it, the more I think the macro/actions-card component is not actually helping us.
There's so much "code" here, and where some of them have markup and href's within them .... I think we'd be better off not using the action card component at all on this page, and just writing the markup in the page.